src/dft.cpp

```diff
@@ -1375,4 +1388,94 @@ void fields::get_mode_mode_overlap(void *mode1_data, void *mode2_data, dft_flux
   get_overlap(mode1_data, mode2_data, flux, 0, overlaps);
 }
 
+// return the size of the dft monitor
+std::vector<int> fields::dft_monitor_size(dft_fields fdft, const volume &where, component c){
```

Comment:
Does `vector<size_t>` work or does SWIG not convert `size_t` to Python correctly?